### PR TITLE
MenuEnterySwapper Dark Mage (Talk - to with Repairs)

### LINKED ISSUE #2806: Menu Swapper for Dark Mage
https://i.imgur.com/uvPVooo.jpg

Move 'Repair' to left click.

### PR DESCRIPTION
**Pull request by KasparRosin**
"closes #2806"
"closes #3464"

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java [hunk 1]
```diff
@@ -355,6 +355,9 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 			{
 				swap("exchange", option, target, true);
 			}
+			if(config.swapDarkMage()){
+				swap("repairs",option,targer,true);
```
Comment (forsco) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java]: Format this line correctly and check spelling.

**Comment by NathenSample** [comment]
Please update the PR description to include
"closes #2806"

**Comment by forsco** [comment]
and also "closes #3464" as that PR is stale.

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java [hunk 2]
```diff
@@ -355,8 +355,9 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 			{
 				swap("exchange", option, target, true);
 			}
-			if(config.swapDarkMage()){
-				swap("repairs",option,targer,true);
+			if (config.swapDarkMage())
+			{
+				swap("repairs",option,target,true);
```
Comment (forsco) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java]: swap("repairs", option, target, true);

**Comment by KasparRosin** [comment]
Anyone has any clue why the check failed?

**Pull request closed without merging**